ibm/mas_devops/roles/kmodels/templates/kmodels-store/deployment.yml.j2

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: km-store
+  namespace: mas-{{ mas_instance_id }}-aibroker
+  labels:
+    app: km-store
+    component: store
+    kmodels: instance
+    part-of: kmodels
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: km-store
+      kmodels: instance
+      part-of: kmodels
+  template:
+    metadata:
+      labels:
+        app: km-store
+        component: store
+        kmodels: instance
+        part-of: kmodels
+    spec:
+      restartPolicy: Always
+      imagePullSecrets:
+        # - name: regcred
+        - name: {{ pullSecretName }}
+      containers:
```

Comment:
is there is no resource request/limits or an liveness/readiness probes>?

Reply:
this is fixed